### PR TITLE
create contact-page branch and update logo

### PR DESCRIPTION
**Pull request by kaka5277**
create contact-page branch,and put logo in it.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -1,511 +0,0 @@
-<!DOCTYPE html>
```
Comment (davidtaing) [inline about.html]: Hey @kaka5277, just double checking. Why were the `about.html` and `index.html` pages deleted?

Once these deletions are checked in, the files will disappear for everyone.

**Review by davidtaing — CHANGES_REQUESTED** [review]
Happy to merge the contact.html page. But need to double check about deleting the other pages.

**Comment by kaka5277** [comment]
hi Dave,i put back the 2 files.i thought only main branch need the files.contact-page branch don't need index.html,about.html

File: contact.html [hunk 2]
```diff
@@ -278,7 +280,7 @@ <h4 class="fw-normal mb-10 color-000">58 Howard St, San Francisco, CA 941</h4>
                                 <a href="home-saas-technology.html"> Home </a>
                             </li>
                             <li>
-                                <a href="page-about-5.html"> About Iteck </a>
+                                <a href="#"> About AICTA </a>
```
Comment (davidtaing) [inline contact.html]: Just need to link this one to the about page.

File: contact.html [hunk 3]
```diff
@@ -258,13 +260,13 @@ <h4 class="fw-normal mb-10 color-000">58 Howard St, San Francisco, CA 941</h4>
...
-                            223 Thatcher Road St, Brookly, <br> Manhattan, NY 10463, US
+                            348 Waterloo Road, GREENACRE, NSW, 2190
                         </small>
                         <small class="text mb-10 d-block">
-                            +031 5689 89 98
+                            +61 417 906 971
                         </small>
                         <small class="text d-block">
-                            contact@Itecksolution.co
+                            <a href="mailto:pleno@bigpond.com" class="color-000"> pleno@bigpond.com </a>
```
Comment (davidtaing) [inline contact.html]: Good work on these!

File: contact.html [hunk 4]
```diff
@@ -64,7 +64,7 @@
     <nav class="navbar navbar-expand-lg navbar-light bg-transparent style-5">
         <div class="container-fluid">
             <a class="navbar-brand" href="#">
```
Comment (davidtaing) [inline contact.html]: The logo should link back to the index page. Instead it will take you to the top of the contact page.

File: contact.html [hunk 5]
```diff
@@ -123,12 +123,12 @@
                 <div class="nav-side">
                     <div class="d-flex align-items-center">
                         <span class="nav-item">
-                            <a class="nav-link" href="page-contact-5.html">
+                            <a class="nav-link" href="#">
                                 <i class="bi bi-person fs-5 me-2"></i>
                                 sign in
                             </a>
                         </span>
```
Comment (davidtaing) [inline contact.html]: Looks like this is a sign in link.

Are there any plans on building an app @engramar or @ZizhangOu? Or is this purely scoped as a static site only.

If it's a static, then we should look at removing this.

File: contact.html [hunk 6]
```diff
@@ -0,0 +1,367 @@
...
+                    <img src="assets/img/icons/contact_a.png" alt="" class="contact_a">
+                    <img src="assets/img/icons/contact_message.png" alt="" class="contact_message">
+                </div>
+            </div>
+        </section>
+        <!-- ====== end contact page ====== -->
+
+        <!-- ====== start contact page ====== -->
+        <div class="map">
+            <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d24219.60999175365!2d-73.9764341314902!3d40.64198229194528!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x89c25b400c94a227%3A0x18e2a4d3fb21f0ec!2sFlatbush%2C%20Brooklyn%2C%20NY%2C%20USA!5e0!3m2!1sen!2seg!4v1651361759450!5m2!1sen!2seg" height="500" style="border:0;" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe>
```
Comment (davidtaing) [inline contact.html:223]: Just need to update this to the Aicta's address.

Let me know if you need help for this one. (I have a pretty good idea of what to do for this one)

**Review by davidtaing — CHANGES_REQUESTED** [review]

File: contact.html (hunk 5 above)
Comment (kaka5277) [reply contact.html]: just updated the links  to their pages,and logo link to home page.

**Comment by davidtaing** [comment]
Good work, @kaka5277. I'll merge this in for now. Let's fix up the map in the next pull request.

**Review by davidtaing — APPROVED** [review]

**Comment by kaka5277** [comment]
thank you dave.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "codesydney/aicta"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年4月6日(星期四) 晚上8:07
***@***.***&gt;;
抄送:&nbsp;"Jun ***@***.******@***.***&gt;;
主题:&nbsp;Re: [codesydney/aicta] create contact-page branch and update logo (PR #12)





 
Merged #12 into main.

**Pull request merged**